src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java

```diff
@@ -113,4 +114,67 @@ public void onReceiveSlashKeeperAddShouldReturnErrorMessageIfOccurException() th
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.text").value("something went wrong"));
     }
+
+    @Test
+    public void onReceiveSlashCommandGetKeeperDirectionsReturnEmptyRichMessage() throws Exception {
```

Comment:
@ksergey12 There is no wiki page yet but you can read this article first. 
@asidun I think we won't have a strict name convention for unit tests. The most idea - the name of test must reveal the contents - what we want to test,  and if we can have several results then the expected result can be mentioned in the name of the test.